### PR TITLE
Add support for LTI 1.3 related directives

### PR DESCRIPTION
**Pull request by PasiSa**
# Description

**What?**

Relates to https://github.com/apluslms/a-plus/pull/1175 and https://github.com/apluslms/gitmanager/pull/34 , and should be merged about at the same time.

**Why?**

[ANSWER HERE]

**How?**

[ANSWER HERE]

Fixes #<issue number>

# Testing

**Remember to add or update unit tests for new features and changes.**

* How to [test your changes in A-plus](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations)
* How to [test accessibility](https://wiki.aalto.fi/display/EDIT/How+to+check+the+accessibility+of+pull+requests)

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [ ] Manual testing.

[ADD A DESCRIPTION ABOUT WHAT YOU TESTED MANUALLY]

**Did you test the changes in**

- [ ] Chrome
- [ ] Firefox
- [ ] This pull request cannot be tested in the browser.

**Think of what is affected by these changes and could become broken**

# Translation

- [ ] Did you modify or add new strings in the user interface? ([Read about how to create translation](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations))

# Programming style

- [ ] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?
- [ ] Did you use Python type hinting in all functions that you added or edited? ([type hints](https://docs.python.org/3/library/typing.html) for function parameters and return values)

# Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [ ] Aplus Manual.
- [ ] Other documentation (mention below which documentation).

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

**Comment by Mankro** [comment]
Does this pull request fix #160?

File: directives/submit.py [hunk 1]
```diff
@@ -90,6 +92,13 @@ def run(self):
                     data.update({'lti_aplus_get_and_post': True})
                 if 'lti_open_in_iframe' in self.options:
                     data.update({'lti_open_in_iframe': True})
+            if 'lti1p3' in self.options:
```
Comment (Mankro) [inline directives/submit.py:95]: You could raise a `SphinxError` if both `lti` and `lti1p3` options are set.

**Review by Mankro — APPROVED** [review]
This looks good, though you could validate incorrect user input better, e.g., mixing options from different LTI versions.

**Comment by Mankro** [comment]
It would also be good to update the README and add a short description of the new options.

File: directives/submit.py [hunk 2]
```diff
@@ -81,6 +83,8 @@ def run(self):
             if 'url' in self.options:
                 data['url'] = self.options['url']
             if 'lti' in self.options:
+                if 'lti1p3' in self.options:
```
Comment (Mankro) [inline directives/submit.py:86]: Should we also check that `lti_custom` is not used with `lti` and `lti_aplus_get_and_post` is not used with `lti1p3`? What about `lti_context_id` and `lti_resource_link_id`?

**Review by Mankro — CHANGES_REQUESTED** [review]
Would you still fix the typos and add the "Fixes #160" text to the git commit message?

**Comment by PasiSa** [comment]
Sorry for my sloppyness, fixed now.

**Review by Mankro — APPROVED** [review]

**Pull request merged**